siriuspy/siriuspy/pwrsupply/psctrl/pscwriters.py

```diff
@@ -366,12 +366,37 @@ def execute(self, value=None):
                 self._cfg.execute(value)
 
 
+class CfgWfm(Function):
+    """Command to configure Wfm."""
+
+    def __init__(self, device_ids, pru_controller, idx, setpoints=None):
+        """Init."""
+        self._idx = idx
+        self._variable_id = _const_psbsmp.V_WFMREF_SELECTED
+        self._setpoints = setpoints
+        self._cfg = BSMPFunction(
+            device_ids, pru_controller, _const_psbsmp.F_CFG_WFMREF)
+
+    def execute(self, value=None):
+        """Execute command."""
+        # get current wfm buffer
+        dev_id = self._cfg.device_ids[0]
+        wfm_selected = self._cfg.pru_controller.read_variables(
+            dev_id, self._variable_id)
+        if not self._setpoints or \
+                (self._setpoints and
+                    self._setpoints.apply(value[self._idx])):
```

Comment:
@xresende, you could start using ruff, as described in this PR #1061, so that we have standardized code.

Reply:
done for this file.